apps/worker/pyproject.toml

```diff
@@ -1,75 +0,0 @@
-[project]
-name = "worker"
-version = "0.1.0"
-description = "The codecov worker"
-readme = "README.md"
-requires-python = ">=3.13"
-dependencies = [
-    "asgiref>=3.7.2",
-    "analytics-python==1.3.0b1",
-    "celery>=5.3.6",
-    "click>=8.1.7",
-    "codecov-ribs==0.1.18",
-    "django>=4.2.17",
-    "django-postgres-extra>=2.0.8",
-    "google-cloud-pubsub>=2.27.1",
-    "httpx>0.23.1",
-    "jinja2>=3.1.5",
-    "lxml>=5.3.0",
-    "mmh3>=5.0.1",
-    "multidict>=6.1.0",
-    "openai>=1.2.4",
-    "orjson>=3.10.11",
-    "polars==1.12.0",
-    "psycopg2-binary>=2.9.10",
-    "pydantic>=2.9.0",
-    "pyjwt>=2.4.0",
-    "python-dateutil>=2.9.0.post0",
-    "python-json-logger>=0.1.11",
-    "python-redis-lock>=4.0.0",
-    "pyyaml>=6.0.1",
-    "redis>=4.4.4",
-    "regex>=2023.12.25",
-    "requests>=2.32.0",
-    "sentry-sdk>=2.29.1",
-    "shared",
-    "sqlalchemy<2",
-    "sqlparse>=0.5.0",
-    "statsd>=3.3.0",
-    "stripe>=11.4.1",
-    "test-results-parser",
-    "timestring",
-    "zstandard>=0.23.0",
-]
-
-[build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools]
-py-modules = []
-
-[tool.uv]
-required-version = ">=0.7.5"
```

Comment:
can you add this as well to the new central pyproject?
as `uv` had quite some churn recently what it is putting into the lockfile, and how those fields are named, we should make sure folks are on an uptodate version.